mypy/nodes.py

```diff
@@ -668,6 +679,11 @@ def deserialize(cls, data: JsonDict) -> 'FuncDef':
         return ret
 
 
+# All types that are both SymbolNodes and FuncBases. See the FuncBase
+# docstring for the rationale.
+SYMBOL_FUNCBASE_TYPES = (OverloadedFuncDef, FuncDef)
```

Comment:
How do we ensure this stays up to date? If a new class were to be added, would tests for that new class fail clearly?

Reply:
They would fail, I think, though I'm not sure how cleanly.